TreasureData/TreasureData.h

```diff
@@ -455,6 +455,38 @@ typedef void (^ErrorHandler)(NSString* _Nonnull errorCode, NSString* _Nullable e
         completionHandler: (void (^_Nonnull)(NSArray* _Nullable jsonResponse, NSError* _Nullable error)) handler
         NS_SWIFT_NAME(fetchUserSegments(tokens:keys:options:completionHandler:));
 
+#pragma mark - Default values
+
+/**
+ * Set default `value` for  `key` in all new events targeting `database` and `table`.
+ * When `database` and/or `table` parameters are nil, the nil parameter acts like a wild card that allows to set specified key value pair to new events added to any database (if `database` is nil) and/or to any table (if `table` is nil).
+ * For example, if you pass nil to both `database` and `table` parameters, all new events will have specified default value.
+ * @param value default value for `key`
+ * @param key the event's key that default value is set to, corresponding to column in table.
+ * @param database the database to set default value to. If nil, specified table of any database will have new events with the added default value.
+ * @param table the table to set default value to. If nil, any table of specified database will have new events with the added default value.
+ */
+- (void)setDefaultValue:(nonnull id)value forKey:(nonnull NSString *)key database:(nullable NSString *)database table:(nullable NSString *)table;
```

Comment:
Provide the same method variants as `addEvent` instead of nullable arguments?

Reply:
Similar to comment in td-android-sdk PR